modules/react/modal/stories/examples/BodyOverflow.tsx

```diff
@@ -18,8 +17,10 @@ export const BodyOverflow = () => {
           <Modal.CloseIcon aria-label="Close" />
           <Modal.Heading>Delete Item</Modal.Heading>
           <Modal.Body>
+            <PrimaryButton>FOCUSSSSSSSSSS</PrimaryButton>
```

Comment:
Remove the test `PrimaryButton`s here (there are a few of them) and the `PrimaryButton` import at the top.